cherab/tools/observers/bolometry.py

```diff
@@ -240,23 +195,19 @@ class BolometerFoil(Node):
     """
 
     def __init__(self, detector_id, centre_point, basis_x, dx, basis_y, dy, slit,
-                 parent=None, render_engine=None):
-
-        self.detector_id = detector_id
+                 parent=None, units="Power", accumulate=False):
 
         # perform validation of input parameters
 
         if not isinstance(dx, float):
             raise TypeError("dx argument for BolometerFoil must be of type float.")
         if not dx > 0:
```

Comment:
I think this is a more important condition than dx being a float. Does passing an integer dx to TargettedPixel break things? Or will it be promoted to float automatically?

Reply:
hmm... yeah I could make it test float and int together. It is important to ensure that the correct type is given, so in this case it could be an int or float. Some objects implement the > and < operators.